### PR TITLE
Bug 1355721 - The value of skipcommit cannot be shown correctly when set router loglevel=4

### PR DESCRIPTION
**Pull request by bmeng**
fix the bug: https://bugzilla.redhat.com/show_bug.cgi?id=1355721

### CODE REVIEW

**Comment by bmeng** [comment]
@knobunc @ramr 
Could you help review?

File: pkg/router/template/router.go [hunk 1]
```diff
@@ -654,7 +654,7 @@ func (r *templateRouter) shouldWriteCerts(cfg *ServiceAliasConfig) bool {
 // commit/reload should be skipped.
 func (r *templateRouter) SetSkipCommit(skipCommit bool) {
 	if r.skipCommit != skipCommit {
-		glog.V(4).Infof("Updating skip commit to: %s", skipCommit)
+		glog.V(4).Infof("Updating skip commit to: %v", skipCommit)
```
Comment (knobunc) [inline pkg/router/template/router.go]: It's a boolean, so let's just use %t here

**Comment by ramr** [comment]
LGTM [test]

**Comment by danmcp** [comment]
[merge]

**Comment by stevekuznetsov** [comment]
re[test]

**Pull request merged**